src/DetectionDetail.cs

```diff
@@ -79,6 +79,17 @@ public DetectionDetail(CharsetProber prober, TimeSpan? time = null)
 
         public string StatusLog { get; set; }
 
+        /// <summary>
+        /// A dictionary for replace unsupported codepage name in .NET to the nearly identical version.
+        /// </summary>
+        /// <remarks>ReadOnlyDictionary start support with .NET Framework 4.5</remarks>
+        private readonly Dictionary<string, string> _fixedToSupportCodepageName =

```

Comment:
Maybe a dictionary is a bit unneeded. What about a `if` in the method above?

Reply:
I would also like to add
> Not supported iso-2022-ch? Maybe fix to x-cp50227?

from #78

It can also be expanded in the future.

Reply:
It's OK if we add those then.

PS I would remove the ReadOnlyDictionary  comment. It's a private.

Reply:
Ready ;)

Reply:
Ow One other thing , this dictionary could be static? As there is no need to recalculate the values multiple times.

Reply:
Yes, I didn’t think about it.